include/SFML/System/Vector2.inl

```diff
@@ -53,6 +53,138 @@ y(static_cast<T>(vector.y))
 }
 
 
+////////////////////////////////////////////////////////////
+template <typename T>
+T Vector2<T>::length() const
+{
+    static_assert(std::is_floating_point_v<T>, "Vector2::length() is only supported for floating point types");
+
+    using std::sqrt; // allow ADL
```

Comment:
Are we doing this in case of a custom `T` used for `Vector2<T>` that has its own sqrt implementation?

Reply:
If so, `std::is_floating_point<T>` would fail. I don't really see the point...

Reply:
I was under the impression that `is_floating_point` can be specialized, but [cppref](https://en.cppreference.com/w/cpp/types/is_floating_point) says:
> The behavior of a program that adds specializations for is_floating_point or is_floating_point_v (since C++17) is undefined. 

So you're right, I'll revert this.